packages/frontend/src/main/components/viewer/ViewerFilters.vue

```diff
@@ -89,6 +79,11 @@
 import { useQuery } from '@vue/apollo-composable'
 import gql from 'graphql-tag'
 import { computed } from 'vue'
+import {
+  getObjectProperties,
```

Comment:
local state values should be read using queries and written to using mutator functions. judging from the name it seems like this reads some values, so this should be moved to the query instead.

if this does in fact mutate the state instead, then the name is very misleading